framework/packages/abstract-std/src/error.rs

```diff
@@ -79,6 +79,9 @@ pub enum AbstractError {
         from: Version,
         to: Version,
     },
+
+    #[error("Migration is not enabled on contract")]
```

Comment:
It's a bit weird that we include this in the abstract-std package when it's only used in standalone, no?

Reply:
Would you prefer it as `generic_err` then? Don't think it's worth adding `StandaloneErr` type just for one error